tests/Browser/TimePicker/view.blade.php

```diff
@@ -12,5 +12,5 @@
     // test it_should_select_time_using_model_property_datetime
     <button dusk="refresh" wire:click="$refresh">$refresh</button>
     <x-time-picker placeholder="12:00" wire:model.defer="user.birthday" label="Model Property" />
-    <span dusk="user.birthday">{{ optional($user->birthday)->format('H:i') }}</span>
+    <span dusk="user.birthday">{{ $user ??= $user->birthday->format('H:i') }}</span>
```

Comment:
```suggestion
    <span dusk="user.birthday">{{ $user?->birthday->format('H:i') }}</span>
```

Reply:
The problem with putting null safe is that the variable will not identify the `format` of something that can be no. The best way to get it is to check if the variable is filled with model.  Best way I found was this instead of checking with If